rxbinding/src/androidTest/java/com/jakewharton/rxbinding/widget/RxAutoCompleteTextViewTestActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.jakewharton.rxbinding.widget;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.widget.ArrayAdapter;
+import android.widget.AutoCompleteTextView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public final class RxAutoCompleteTextViewTestActivity extends Activity {
+  AutoCompleteTextView autoCompleteTextView;
+
+  List<String> values;
+  ArrayAdapter<String> adapter;
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    values = new ArrayList<>(Arrays.asList("One", "Two", "Three"));
```

Comment:
You don't need `new ArrayList`. Just setting the value to `Arrays.asList` is enough